From d623d98fc42cd7dcacb397ecdf5cd3e747647376 Mon Sep 17 00:00:00 2001
From: hluwa <hluwa888@gmail.com>
Date: Thu, 25 Mar 2021 22:47:38 +0800
Subject: [PATCH 2/8] strongR-frida: io_re_frida_server

---
 server/server.vala | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/server/server.vala b/server/server.vala
index 0b996e96..8dd710e3 100644
--- a/server/server.vala
+++ b/server/server.vala
@@ -3,7 +3,7 @@ namespace Frida.Server {
 
 	private const string DEFAULT_LISTEN_ADDRESS = "127.0.0.1";
 	private const uint16 DEFAULT_LISTEN_PORT = 27042;
-	private const string DEFAULT_DIRECTORY = "re.frida.server";
+	private static string DEFAULT_DIRECTORY = null;
 	private static bool output_version = false;
 	private static string? listen_address = null;
 	private static string? directory = null;
@@ -38,6 +38,7 @@ namespace Frida.Server {
 	};
 
 	private static int main (string[] args) {
+		DEFAULT_DIRECTORY = GLib.Uuid.string_random();
 		Environment.init ();
 
 #if DARWIN
-- 
2.28.0

