From 71d916705f3694adbd51de8b54bbd26c5697767d Mon Sep 17 00:00:00 2001
From: hluwa <hluwa888@gmail.com>
Date: Sat, 19 Dec 2020 01:43:38 +0800
Subject: [PATCH 7/8] thread_gmain

---
 src/anti-anti-frida.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/anti-anti-frida.py b/src/anti-anti-frida.py
index 785075ca..a6fa3f3a 100644
--- a/src/anti-anti-frida.py
+++ b/src/anti-anti-frida.py
@@ -26,3 +26,9 @@ if __name__ == "__main__":
     random_name = "".join(random.sample("abcdefghijklmn", 11))
     print(f"[*] Patch `gum-js-loop` to `{random_name}`")
     os.system(f"sed -b -i s/gum-js-loop/{random_name}/g {input_file}")
+
+
+    # gmain thread
+    random_name = "".join(random.sample("abcdefghijklmn", 5))
+    print(f"[*] Patch `gmain` to `{random_name}`")
+    os.system(f"sed -b -i s/gmain/{random_name}/g {input_file}")
\ No newline at end of file
-- 
2.28.0

